src/main/java/com/northcoders/makemydayapi/service/RestaurantService.java

```diff
@@ -0,0 +1,130 @@
+package com.northcoders.makemydayapi.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.northcoders.makemydayapi.model.GeoapifyRestaurant;
+import com.northcoders.makemydayapi.model.RestaurantType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Service
+public class RestaurantService {
```

Comment:
Can we keep service as a more generic `GeoapifyService` and have different methods of `getRestaurants`, `getParks`, etc., instead of "one service for one sub-activity type"?

It would be more manageable for us since we got lots of different sub-activity types from 3 different public APIs.